### PR TITLE
Add Technician playbook

### PR DESCRIPTION
**Pull request by flyinggrizzly**
This PR adds the auto-gen data for the Technician playbook:

- [x] Description
- [x] XP trigger
- [x] Load/gear
- [x] Vehicle load/gear
- [x] Abilities
- [x] Starting action ratings
- [x] Gather info methods
- [x] Friends

References #3 

Screen Shot 2019-03-23 at 13 59 20
Screen Shot 2019-03-23 at 13 59 31

### CODE REVIEW

File: translation.json [hunk 1]
```diff
@@ -327,19 +335,19 @@
...
     "playbook_ace_xp_condition": "You addressed a challenge with piloting or violence.",
+    "playbook_technician_description": "A savvy\ntechnician and\nvandal.",
+    "playbook_technician_xp_condition": "You addressed a challenge with technical skill or mayhem.",
     "playbook_item_grav_chute": "Grav chute",
     "playbook_item_fine_remote_vehicle_control": "Fine remote vehicle control",
     "playbook_item_self_destruct_remote": "Self destruct remote",
     "playbook_item_fine_environmental_suit_wearing": "Fine environmental suit (W)",
     "playbook_item_fine_environmental_suit_carried": "Fine environmental suit (C)",
     "playbook_item_your_custom_vehicle": "Your custom vehicle",
+    "playbook_item_fine_engineering_tools": "Expertly made tools used for bine mechanical work.",
```
Comment (Solarswordsman) [inline translation.json]: The name of the item (`playbook_item_fine_engineering_tools`) should just be its name: "Fine Engineering Tools"

To add a description, add one in the `playbookitem` key with the key `description` e.g.
```
"bold": "1",
"name": "playbook_item_fine_engineering_tools",
"description": "playbook_item_fine_engineering_tools_description",
"numboxes": "1"
```

File: translation.json [hunk 2]
```diff
@@ -367,6 +375,7 @@
     "playbook_vehicle_item_fine_machine_gun": "Fine Machine Gun",
     "playbook_vehicle_item_fine_shield_or_forcefield": "Fine Shield OR Forcefield",
     "playbook_vehicle_item_rage": "R.A.G.E.",
+    "playbook_vehicle_item_fine_modding_tools": "Excellent tools for modifying large objects on the fly.",
```
Comment (Solarswordsman) [inline translation.json]: Same here as the `playbookitem`.

**Review by Solarswordsman — CHANGES_REQUESTED** [review]
Description for items should be a separate key, and specified as "description" in the item's data object.

I didn't add any description for playbook specific items on Ace, but if present/added they should work.

File: translation.json (hunk 1 above)
Comment (flyinggrizzly) [reply translation.json]: Thanks for the heads up--fixed these and done it right for the rest of the gear.

**Comment by flyinggrizzly** [comment]
@Solarswordsman this should be good, other than the issue raised in #7.

I was thinking I'd squash it down to just three commits, one per file. Let me know if you when that's fine by you (I'm finding it a lot easier to go through this commit by commit since there's so much).

**Comment by Solarswordsman** [comment]
Sure, that's fine.  Alternatively I can just squash-merge it to keep master's history cleaner.

**Comment by flyinggrizzly** [comment]
@Solarswordsman should be ready to go now. Squashed down to a commit per file. If you want to squash-merge on top of that that's cool too 👍

**Review by Solarswordsman — APPROVED** [review]

**Pull request merged**